libglusterfs/src/glusterfs/inode.h

```diff
@@ -111,9 +111,11 @@ struct _inode {
     struct list_head hash;        /* hash table pointers */
     struct list_head list;        /* active/lru/purge */
 
+    struct _inode *ns_inode; /* This inode would point to namespace inode */
     struct _inode_ctx *_ctx; /* replacement for dict_t *(inode->ctx) */
     bool in_invalidate_list; /* Set if inode is in table invalidate list */
     bool invalidate_sent;    /* Set it if invalidator_fn is called for inode */
+    gf_atomic_t kids;
```

Comment:
Perhaps worth aligning, before the bool params above

Reply:
Done